roles/content_views/tasks/_create_content_view.yml

```
@@ -9,7 +9,7 @@
     name: "{{ content_view.name }}"
     auto_publish: "{{ content_view.auto_publish | default(omit) }}"
     components: "{{ content_view.components | default(omit) }}"
-    composite: "{{ content_view.components | default(false) | ternary(true, false) }}"
+    composite: "{{ content_view.composite | default(false) | ternary(true, false) }}"
```

```suggestion
    composite: "{{ content_view.components | default(content_view.composite) | default(false) | ternary(true, false) }}"
```

does that work for your "empty CCV" case?

---

Give me like 4h and I'll be able to tell you. Thanks!!

---

It's the same code I've ended up using after my re-thought of the issue, so big +1 from me :)